specs/~inheritance.yml

```diff
@@ -224,3 +224,92 @@ tests:
     partials:
       parent: "{{$foo}}default content{{/foo}}"
     expected: default content
+
+  - name: Standalone parent
+    desc: A parent's opening and closing tags need not be on separate lines in order to be standalone
+    data: {}
+    template: |
+      Hi,
+        {{<parent}}{{/parent}}
+    partials:
+      parent: |
+        one
+        two
+    expected: |
+      Hi,
+        one
+        two
+
+  - name: Standalone block
+    desc: A block's opening and closing tags need not be on separate lines in order to be standalone
```

Comment:
This is however not exactly the same as a single standalone partial tag, because that would conflict with the existing "Inherit indentation" spec on lines 154-164. "Standalone" here mostly refers to whether the block as a whole will be indented. Contrary to the previous spec, if the end section tag is not standalone by itself, then the trailing newline will **not** be removed from the output if the block ends up being empty.